### DISCUSSION TITLE
（总结帖）对于类似的线上维权，大家有什么可以总结的经验告诉后人吗？

### DISCUSSION THREAD

**Discussion by richardo876** [WeNeedHome/SummaryOfLoanSuspension#984] [General]
和996.icu项目类似，我姑且称这类维权叫“去中心线上维权”。像996和这个项目到底为什么“失败”？ 我们维权的目的达到了吗？ 如果没有达到，有什么需要改进的地方吗？ 
（趁现在大家还没跑光，大家集思广益，来给点对后人的敬言哈）

**Comment by richardo876** [comment]
我抛砖引玉说一个：我认为**诉求要清晰**。这个项目从头到尾诉求不清，所谓统计停贷数据到底为什么要统计？这个东西都没说清楚，导致媒体想转发报导都不知从何说起。

**Reply by xiejun946** [reply]
停贷这个项目只需要列出数据，做出行动就好了。后来者需要的是开发商的数据，楼盘品质，资金状态，如果一定要买预售楼房，需要了解这些

**Reply by solidusxin** [reply]
就算有诉求也是一样，河南银行就是例子。政府不喜欢的事，注定一开始就要以一个政府无法估量的体量迅速铺过去，类似唐山案件，这样政府不得不面对，处理。至于之后的，至少在处理了总比不闻不问要好。剩下的就真不知道了

**Comment by CaiDeng** [comment]
诉求不需要清晰，靠大家自己琢磨被，毕竟都是聪明人

**Reply by richardo876** [reply]
那为啥还要统计停贷数据了，还不如各个楼盘自己琢磨？

**Reply by CaiDeng** [reply]
![image](https://user-images.githubusercontent.com/28388777/180447755-15c77e83-9b5d-4983-803a-53bd42b5c56d.png)

**Reply by richardo876** [reply]
哪句说了“为什么要统计？”

**Reply by iosyyy** [reply]
> 哪句说了“为什么要统计？”

这不很明显是为了烂尾停贷这些人的利益吗? 统计当然是给上面的人看的 人多了上面当然就有措施了 另外不是媒体想转发报道没办法而是他们和银行政府就是一伙 你去国内看看这话题的热度早就封没了和shga一样

**Reply by richardo876** [reply]
> > 哪句说了“为什么要统计？”
> 
> 这不很明显是为了烂尾停贷这些人的利益吗? 统计当然是给上面的人看的 人多了上面当然就有措施了 另外不是媒体想转发报道没办法而是他们和银行政府就是一伙 你去国内看看这话题的热度早就封没了和shga一样

我说的媒体当然说的是不受ccp管制的媒体。

**Comment by iosyyy** [comment]
只能是要么你还年轻要么没经历很多 这种事和河南乃事一样根本不能正常维权 正常维权只能是银行和政府相互踢皮球

**Reply by richardo876** [reply]
你说的点是对的。 但没有回答“如何改进线上维权”这个问题。  （By the way，翻墙维权本来就不是正常维权）

**Reply by xiejun946** [reply]
我觉得没办法改进。破罐子破摔，号召以后大家抵制预售制，是唯一能做的。

**Comment by TALK1949** [comment]
油管有位播主，总结过方法，但是因为在我看来容易碰触到某些人的玻璃心，就没放。
她的观点简单来说就是，抓住银行违规的把柄，也有详细说明哪些把柄可以抓以及怎么抓，总体而言对普通人来说很难，当然这说的是“正常”途径。
https://youtu.be/eYHvsVGfJTg?t=393

**Reply by CaiDeng** [reply]
这个是有点用，最近不是出了银行和房管局在互相踢皮球么

**Reply by xiejun946** [reply]
非法律金融专业我觉得都不懂

**Comment by volans19** [comment]
没有后人了，我们是最后一代人

**Reply by icm-ai** [reply]
> 没有后人了，我们是最后一代人

后面的都是奴隶？

**Reply by wedxz** [reply]
> 

连孩子都没人生了,哪还有后面
